pallets/parachain-staking/src/lib.rs

```diff
@@ -392,6 +377,11 @@ pub mod pallet {
         }
     }
 
+    #[pallet::storage]
+    #[pallet::getter(fn delay)]
+    /// Number of eras to wait before executing any staking action
+    pub type Delay<T: Config> = StorageValue<_, EraIndex, ValueQuery>;
```

Comment:
I think we are missing a TODO here, to set the value on existing chains via a storage upgrade

Reply:
There is a separate ticket for that.